src/components/Switcher/Switcher.js

```diff
@@ -18,22 +18,20 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 import * as React from 'react';
-import { Animated, TouchableOpacity } from 'react-native';
-import styled, { withTheme } from 'styled-components/native';
-import type { Theme } from 'models/Theme';
-import { getThemeColors } from 'utils/themes';
+import { Animated } from 'react-native';
+import styled, { css } from 'styled-components/native';
+import { getColorByTheme } from 'utils/themes';
 import { LIGHT_THEME } from 'constants/appSettingsConstants';
 
 type Props = {
-  isOn: boolean,
-  onToggle: () => void,
-  disabled?: boolean,
-  theme: Theme,
-}
+  isOn: ?boolean,
+  onToggle: ?() => void,
+  disabled?: ?boolean,
```

Comment:
It might be either present boolean or nothing, right?
```suggestion
  disabled?: boolean,
```